mlserver/grpc/converters.py

```diff
@@ -135,6 +136,12 @@ def to_types(cls, pb_object: pb.ModelInferRequest) -> types.InferenceRequest:
                 for out in pb_object.outputs
             ]
 
+        if pb_object.raw_input_contents:
+            # Unpack and inject raw contents into `data` fields if present
+            inject_raw(
+                inference_request.inputs, pb_object.raw_input_contents  # type: ignore
```

Comment:
it wasnt very clear that grpc takes care of returning a list of bytes in `raw_input_contents` one for each input request. I thought initially this is one big stream of bytes but maybe it is only me.

anyway perhaps add a note to make it clearer?

Reply:
ok  fine if there is a note somewhere else.